authentik/core/tests/test_devices_api.py

```diff
@@ -0,0 +1,57 @@
+"""Test Devices API"""
+
+from json import loads
+
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+from authentik.core.tests.utils import create_test_admin_user, create_test_user
+
+
+class TestApplicationsAPI(APITestCase):
```

Comment:
```suggestion
class TestDevicesAPI(APITestCase):
```

Reply:
I don't know what you mean, i absolutely didn't copy other tests and modify them 😇